src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeCustomAttributeData.cs

```diff
@@ -1114,7 +1133,7 @@ public CustomAttributeType(RuntimeType parameterType)
         public Type? EnumType { get; }
     }
 
-    internal static unsafe class CustomAttribute
+    internal static unsafe partial class CustomAttribute
```

Comment:
```suggestion
    internal static unsafe class CustomAttribute
```
I do not see why this needs to be partial.

Reply:
This is needed because of the use of `LibraryImport` below.